onnxruntime/python/tools/quantization/data_preprocess.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+# coding: utf-8
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft, Intel Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+from PIL import Image
+import os
+import sys
+import numpy as np
+
+def set_preprocess(preprocess_func_name):
+    '''
+    Set up the data preprocess function name and function dict. 
+        parameter preprocess_func_name: name of the preprocess function 
+        return: function pointer 
+    '''
+    funcdict = {'preprocess_method1': preprocess_method1, 
+                'preprocess_method2': preprocess_method2}
+    return funcdict[preprocess_func_name]
+
+def preprocess_method1(image_filepath, height, width):
+    '''
+    Resizes image to NCHW format. Image is scaled to range [-1, 1].
+    This method is suitable for the mobilenet model from mlperf inference git repo.
+        parameter image_filepath: path to image files
+        parameter height: image height in pixels
+        parameter width: image width in pixels
+        return: matrix characterizing image
+    '''
+    pillow_img = Image.open(image_filepath).resize((width, height))
+    input_data = np.float32(pillow_img)/127.5 - 1.0 # normalization
+    input_data -= np.mean(input_data) # normalization
+    nhwc_data = np.expand_dims(input_data, axis=0)
+    nchw_data = nhwc_data.transpose(0, 3, 1, 2) # ONNX Runtime standard
+    return nchw_data
+
+def preprocess_method2(image_filepath, height, width):
```

Comment:
give a name which better describes this pre-processing method and same for "preprocess_method1" too

Reply:
Will do it.